zedd-app/src/Undoer.ts

```diff
@@ -40,6 +40,7 @@ export class Undoer {
                 removed: change.removed,
                 index: change.index,
                 element: change.object,
+                timeStamp: Math.floor(Date.now() / 10),
```

Comment:
take "Math.floor(Date.now() / 10)" to a const variable and use it every time you assign timeStamp